ruby/ql/lib/codeql/ruby/security/IncompleteMultiCharacterSanitizationQuery.qll

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
I think it should be possible to share most of this file with the Javascript variant.